hardware/schematic.md

```diff
@@ -85,13 +85,11 @@ Some of the Columns appear on the edge connector, so if you want to use extra GP
 
 ### Interface
 
-The Interface sheet shows the KL26<span class="v1">V1</span>/KL27<span class="v2">V2</span> processor, which is an NXP microcontroller with an Arm processor, that implements the USB protocol for the USB connector. This provides a method for loading code onto the application processor, using a drag and drop interface.
+The Interface sheet shows the KL26<span class="v1">V1</span>/ KL27<span class="v2">V2.0</span> / nRF52<span class="v2">V2.2</span> processor, which is an NXP or Nordic microcontroller with an Arm processor, that implements the USB protocol for the USB connector. This provides a method for loading code onto the application processor, using a drag and drop interface.
```

Comment:
```suggestion
The Interface sheet shows the KL26<span class="v1">V1</span> / KL27<span class="v2">V2.0</span> / nRF52<span class="v2">V2.2</span> processor, which is an NXP or Nordic microcontroller with an Arm processor, that implements the USB protocol for the USB connector. This provides a method for loading code onto the application processor, using a drag and drop interface.
```

Reply:
I might be miss-remembering it (very possible), but I think I didn't add the space between the `span` and the `/` because the `.v1` class has some padding and it rendered better without it? did you check how this paragraph is rendered?

Reply:
With these docs, this is how it renders. I don't have a preference, but it's not consistent across the two instances of a version class and "/".

![image](https://user-images.githubusercontent.com/95928279/202170084-d08a94dd-a379-440e-866a-36c8d46ca440.png)

Reply:
Cool, added in force push https://github.com/microbit-foundation/dev-docs/compare/b96c9a28afef09f7aee657bd4f43ec641afa0d3a..400811505ff84b6d6402dc6ccbd92c1283666b27